readme.md

```diff
@@ -1,31 +1,32 @@
-# Haunted 🦇 🎃
+🦇 🎃
 
 [![npm](https://img.shields.io/npm/dt/haunted)](https://npm.im/haunted)
 [![npm](https://img.shields.io/npm/v/haunted)](https://npm.im/haunted)
 
 React's Hooks API but for standard web components and [lit-html](https://lit-html.polymer-project.org/) or [hyperHTML](https://codepen.io/WebReflection/pen/pxXrdy?editors=0010).
 
 ```html
-<!doctype html>
```

Comment:
This is usually wanted, why remove this?

Reply:
It's usually wanted for a full web page, as are things like `<head>`, `<title>`, and `<body>` but they're unnecessary for just the bare example that's being given here

Reply:
I can add it back in if the example is better with it.